packages/server/src/hono/routes/a2a-tasks.ts

```
@@ -0,0 +1,365 @@
+        'working',
+        'input-required',
+        'completed',
+        'canceled',
+        'failed',
+        'rejected',
+        'auth-required',
+        'unknown',
+    ]),
+    message: MessageSchema.optional(),
+    timestamp: z.string().optional(),
+});
```

ESLint

[error] Zod method chain is missing .describe() call. Add .describe("description here") for OpenAPI documentation.

(dexto-custom/require-zod-describe)